requirements.txt

```diff
@@ -1,4 +1,4 @@
 PySide2
 requests
 psutil
-RPI.GPIO
\ No newline at end of file
+RPI.GPIO; sys_platform == 'linux'
```

Comment:
This is a good enhancement, but not all linuxes are raspberry pi os. According to https://peps.python.org/pep-0508/#environment-markers we should also be able to limit the `platform_machine` - to `armv7l` - which would further narrow it down.

Reply:
Good call, I'll dig in and see if we can grab some additional values to check as (iirc) there's also an ARM64 version of piOS.